spec/controllers/adjustments_controller_spec.rb

```diff
@@ -0,0 +1,82 @@
+RSpec.describe AdjustmentsController, type: :controller do
```

Comment:
We generally prefer request specs over controller specs (see [https://github.com/rspec/rspec-rails/?tab=readme-ov-file#request-specs](https://github.com/rspec/rspec-rails/?tab=readme-ov-file#request-specs) ). Can this be moved?

Reply:
thanks for the pointer, will change in the next commits, thanks :)